src/customHooks/useTodoChange.tsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable max-len */
+import { useState } from 'react';
+
+export const useTodoChange = (): [number | number[], boolean, ((changingTodosId: number | number[], state: boolean) => void)] => {
```

Comment:
please don't over-stretch the lines - it's really hard to read
```suggestion
export const useTodoChange = (): [
	number | number[],
	boolean,
	((changingTodosId: number | number[], state: boolean) => void),
] => {
```